internal/node-patches/prettier.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  singleQuote: true,
```

Comment:
I would expect formatter options to be shared across the whole monorepo
(we already have a clang-format setup at the root, we could change to prettier in a separate PR)

Reply:
not in scope. this is generated by gts

Reply:
I think it is in scope, this will have strange interactions with the monorepo-wide formatter, let's remove from the PR please